eng/pipelines/templates/jobs/tests-nightly-python.yml

```
@@ -130,7 +130,11 @@ jobs:
           export PATH=~/.local/bin:$PATH
           curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
           python3 get-pip.py
-          python3 -m pip install setuptools wheel
+          python3 -m pip install 
+          
+          
+          
+          wheel
```

Not sure if this suggestion is what we're going for, but it this line seems to have gotten confused
```suggestion
          python3 -m pip install setuptools==67.6.0 wheel
```

---

Saving me from myself @mccoyp 

I have no idea how this even made it into the changeset.